tests/cypress/support/commands_masks.js

```diff
@@ -0,0 +1,65 @@
+// Copyright (C) 2023 CVAT.ai Corporation
+//
+// SPDX-License-Identifier: MIT
+
+Cypress.Commands.add('drawMask', (instructions) => {
```

Comment:
You created a dedicated commands file for masks, I do not mind, but we need some generalized approach here.
Why do not we have in this case dedicated files for skeletons, rectangles, etc?

Reply:
I've noticed that our main `commands.js` file gets way too big with adding such functions. What do you think about moving shapes commands in separate `commands_shapes.js`?

Reply:
Need to think better and probably it is a thing for a separate PR. Because now we have spaghetti here, for example command to draw a 3D cuboid shape is located in ``commands_canvas3d``

Reply:
Okay, lets keep that improvement for separate pr